7_Crypto Analysis.py

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+
+# Crypto Analysis: Frequency Analysis
+#
+# To analyze encrypted messages, to find out information about the possible 
+# algorithm or even language of the clear text message, one could perform 
+# frequency analysis. This process could be described as simply counting 
+# the number of times a certain symbol occurs in the given text. 
+# For example:
+# For the text "test" the frequency of 'e' is 1, 's' is 1 and 't' is 2.
+#
+# The input to the function will be an encrypted body of text that only contains 
+# the lowercase letters a-z. 
+# As output you should return a list of the normalized frequency 
+# for each of the letters a-z. 
+# The normalized frequency is simply the number of occurrences, i, 
+# divided by the total number of characters in the message, n.
+
+def freq_analysis(message):
+  def freq_analysis(message):
+  totalnum=len(message)
+  freqList=[]
+  import string
```

Comment:
why an import statement here?

Reply:
i had to import string to use asciii_lowercase. but u are right i should have placed outside function body